src/ui/GenioWindow.cpp

```diff
@@ -3524,16 +3523,28 @@ GenioWindow::_ProjectFolderOpen(const entry_ref& ref, bool activate)
 	if (BDirectory(&dirEntry).IsRootDirectory())
 		return B_ERROR;
 
-	// Check if already open
+	BPath newProjectPath;
+	status_t status = dirEntry.GetPath(&newProjectPath);
+	if (status != B_OK)
+		return status;
 	for (int32 index = 0; index < GetProjectBrowser()->CountProjects(); index++) {
 		ProjectFolder* pProject = GetProjectBrowser()->ProjectAt(index);
+		const BString existingProjectPath = pProject->Path();
+		// Check if it's a subfolder of an existing open project
+		// TODO: Ideally, this wouldn't be a problem: it should be perfectly possibile
+		if (BString(newProjectPath.Path()).StartsWith(existingProjectPath))
```

Comment:
Maybe we should add to the existingProjectPath and to the newProjectPath a trailing "/" to avoid some wrong matching..

Reply:
BPath does this already, but we are using BString so you're right

Reply:
Maybe adding / to the existingProjectPath should be sufficient, though.